tests/unit/__snapshots__/open-api.spec.ts.snap

```diff
@@ -2345,6 +2347,440 @@ servers:
 "
 `;
 
+exports[`Open API generator generateOpenApi() should generate the correct schema of example routing 1 1`] = `
+"openapi: 3.0.0
+info:
+  title: Example API
+  version: 1.2.3
+paths:
+  /v1/user/retrieve:
+    get:
+      operationId: GetV1UserRetrieve
+      responses:
+        "200":
+          description: GET /v1/user/retrieve Successful response
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/GetV1UserRetrieveSuccessfulResponse"
+        "400":
+          description: GET /v1/user/retrieve Error response
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/GetV1UserRetrieveErrorResponse"
+              examples:
+                example1:
+                  value:
+                    status: error
+                    error:
+                      message: Sample error message
```

Comment:
in the future may utilize the same approach for extracting examples into components in case of this composition